app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Just had a doubt on this part @rt4914 will address this if needed rest looks fine to me